lab_1/stop_words.txt

```diff
@@ -71,6 +71,7 @@ better
 between
 beyond
 both
+br
```

Comment:
I do not understand this addition. Can you explain?

Reply:
That was for the case if one of tests (marked with @unittestskip or so) starts working. I guess I should delete it

Reply:
That's true. We removed that test.